tst/unit/test_data_collection.cpp

```diff
@@ -97,7 +97,9 @@ TEST_CASE("Adding MeshBlockData objects to a DataCollection", "[DataCollection]"
           KOKKOS_LAMBDA(const int i) { v2(0) = 222; });
       auto x = d.Add("part", mbd, {"var2", "var3"});
       THEN("Requesting the missing variables should throw") {
-        REQUIRE_THROWS(x->Get("var1"));
+        // This no longer call PARTHENON_REQUIRE_THROWS, it just call PARTHENON_REQUIRE
+        // since throwing does not work nicely with threads
+        // REQUIRE_THROWS(x->Get("var1"));
```

Comment:
Not for this PR, but this should be fixed. See, e.g., issue #1046 .

Reply:
I know all too well... don't ask me why this PR took a week to get done instead of a day and a half.

Reply:
Should this be reintroduced now?

Reply:
yep. It is now reintroduced.